aria-practices.html

```diff
@@ -3734,6 +3734,390 @@ <h5>Examples</h5>
+
+        <p>
+          When this attribute is omitted, the user agent will use the closest ancestor that has <code>aria-atomic</code> set to <code>true</code> or <code>false</code>,
+          or if there is no such ancestor, <code>false</code>.
+        </p>
+
+        <p>
+          For example, consider a clock that can be set to notify the user of the current time at a regular interval.
+          Without <code>aria-atomic</code>, the assistive technology might only notify the changed components, rather than the full time.
+        </p>
+
+        <pre><code>&lt;div id="clock" role="region" aria-live="polite" aria-atomic="true">
+ The time is
+ &lt;span>16&lt;/span>:&lt;span>34&lt;/span>:&lt;span>05&lt;/span>
+&lt;/div>
+</code></pre>
+      </section>
+
+      <section id="indicate_which_content_changes_are_relevant_with_aria-relevant">
+        <h4>Indicate Which Content Changes are Relevant with <code>aria-relevant</code></h4>
+
+        <p>
+          The <code>aria-relevant</code> attribute can be used to inform assistive technologies about which kinds of changes are relevant to inform users about.
+          It takes a list of keywords, with the following meanings:
+        </p>
+
+        <ul>
+          <li><code>additions</code>: Element nodes are added to the accessibility tree within the live region.</li>
+          <li><code>text</code>: Text content or a text alternative is added to any descendant in the accessibility tree of the live region.</li>
+          <li><code>removals</code>: Text content, a text alternative, or an element node within the live region is removed from the accessibility tree.</li>
+          <li><code>all</code>: Synonym to <code>additions removals text</code></li>
+        </ul>
+
+        <p>
+          If <code>aria-relevant</code> is not specified, then the value of the closest ancestor element with an <code>aria-relevant</code> attribute is used.
+          Specifying the <code>aria-relevant</code> attribute on an element overrides any value specified on an ancestor element.
+          If there is no ancestor element with an <code>aria-relevant</code> attribute, the default value <code>additions text</code> is used.
+        </p>
+
+        <p>
+          For example, a disappearing old message in a chat log is not significant, and users do not need to be informed of the removal.
+          However, for a list of online contacts, a disappearing contact is significant (it indicates that the contact is no longer online).
+          Instead of announcing the removal of something, it is often better to add new content that tells the user what happened.
+          For example, "Alice is now offline" is clearer than "Alice" or "Removed, Alice".
+          Therefore, avoid using <code>removals</code> or <code>all</code>.
+        </p>
+
+        <p>
+          For example, a list of online contacts:
+        </p>
+
+        <pre><code>&lt;div role="region" aria-live="polite" aria-labelledby="contacts">
```

Comment:
See note above regarding `role="region"`. Note that in this particular example, since the `region` has a name ("Contacts") it will definitely be a landmark. For this particular example, this is probably what the author wants - it makes sense to have the contacts list be a landmark, therefore easily navigable. I'm just flagging this to include it in the discussion.